monkey/monkey_island/cc/resources/pba_file_download.py

```diff
@@ -0,0 +1,14 @@
+import json
+import logging
+import os
+
+import flask_restful
+from flask import request, send_from_directory
+
+UPLOADS_DIR = "./userUploads"
```

Comment:
For now I make a constant that uses relative path in every file where I need to reference this dir. Feels like there is a better way...